### DISCUSSION TITLE
Where come the values for Model, etc.

### DISCUSSION THREAD

**Discussion by usauerbrey** [edterbak/HeishaMoNR#103] [Q&A] [answered]
In the Settings under Panasonic, info of Model, Type, Heat capacity, etc are shown.

But where do these values come from, HeishaMon does it not provide via MQTT, at least are these value not shown on the Heisha monitor page

**Comment by edterbak** [comment, marked as answer]
TOP 92 is the number which is translated to the model. 
There are newer versions available I see, so I will update the translation table soon.
The translationtable you can find in the WP Control tab, towards the bottom in the function node [Heatpump type (Panasonic)].
You can edit it in the mean time.

**Comment by edterbak** [comment]
Ah, sorry, I stand corrected.
Current node red flow holds models up to no 28. Which is last known model number to date
